### PR TITLE
Update dialog to CDS styles

### PR DESCRIPTION
**Pull request by brianfleming**
## Description

[A previous PR](https://github.com/meedan/check-web/pull/1723) introduced some integration test failures because of changes to confirmation dialog dom elements. This PR fixes those failures and updates some other dialogs to consistent styles

Also
- Updates some material components for CDS while I was in there
- added in missing tooltips for the new chat history buttons on the requests lists
- replaced some old circular progress components for meedan custom loading indicator
- added the ability to show or hide the label for workspace permissions (roles) depending where the select is used

## Type of change

- [ ] Performance improvement and/or refactoring (non-breaking change that keeps existing functionality)
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Security mitigation or enhancement
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Automated test (add or update automated tests)

## How has this been tested?

Running integration tests on CI

## Things to pay attention to during code review

nada

## Checklist

- [x] I have performed a self-review of my own code
- [x] I've made sure my branch is runnable and given good testing steps in the PR description
- [ ] I considered secure coding practices when writing this code. Any security concerns are noted above.
- [x] I have commented my code in hard-to-understand areas, if any
- [ ] I have made needed changes to the README
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] If I implemented any new components, they are self-contained, their `propTypes` are declared and they use React Hooks and, if data-fetching is required, they use Relay Modern with fragment containers
- [x] If my components involve user interaction - specifically button, text fields, or other inputs - I have added a [BEM-like class name](https://meedan.atlassian.net/wiki/spaces/ENG/pages/1327628289/Naming+conventions+for+interactive+elements) to the element that is interacted with
- [x] To the best of my knowledge, any new styles are applied according to the design system
- [ ] If I added a new external dependency, I included a rationale for doing so and an estimate of the change in bundle size (e.g., checked in https://bundlephobia.com/)

### CODE REVIEW

File: src/app/components/layout/ConfirmProceedDialog.js [hunk 1]
```diff
@@ -78,7 +78,9 @@ const ConfirmProceedDialog = ({
             label={cancelLabel || <FormattedGlobalMessage messageKey="cancel" />}
           /> : null }
         <ButtonMain
-          id="confirm-dialog__confirm-action-button"
+          buttonProps={{
```
Comment (brianfleming) [inline src/app/components/layout/ConfirmProceedDialog.js:81]: This was the main reason for the test failures. I incorrectly was setting the id, which was not getting to the dom as originally coded

File: src/app/components/SendTiplineMessage.module.css
Comment (brianfleming) [inline src/app/components/SendTiplineMessage.module.css:1]: I folded the close button into the generic dialog stylesheet, so this file is no longer necessary

File: src/app/components/team/RoleSelect.js [hunk 2]
```diff
@@ -41,40 +37,35 @@ const RoleSelect = ({
...
+      }
+    </Select>
   );
 };
 
 RoleSelect.defaultProps = {
   disabled: false,
   excludeRoles: [],
-  fullWidth: false,
+  showLabel: false,
```
Comment (brianfleming) [inline src/app/components/team/RoleSelect.js:62]: No need for the full width anymore, but this show label is to hide the label when in the table view

File: test/spec/similarity_spec.rb [hunk 3]
```diff
@@ -68,9 +68,9 @@
     wait_for_selector('#media-similarity__add-button')
     expect(@driver.page_source.include?('claim 2')).to be(false)
     wait_for_selector("//span[contains(text(), 'Suggestions')]", :xpath).click
-    wait_for_selector("//span[contains(text(), '2 Suggestions')]", :xpath)
+    wait_for_selector("//span[contains(text(), '/ 2')]", :xpath)
```
Comment (brianfleming) [inline test/spec/similarity_spec.rb:71]: Pager was changed to the same style as the main lists, without this extra text

**Comment by brianfleming** [comment]
Ok, this is ready to go I think. The last run of integration tests failed with those typical 4 failures. The others have passed now.

**Review by amoedoamorim — APPROVED** [review]

**Pull request merged**